sentry2csv/sentry2csv.py

```diff
@@ -45,15 +47,18 @@ async def fetch(
     async with session.get(url, params=params) as response:
         logger.debug("Received response: %s", response)
         if response.status == 403:
-            raise Sentry2CSVException(f"Failed to query Sentry: access denied.")
+            raise Sentry2CSVException("Failed to query Sentry: access denied.")
         return await response.json(), response.links
 
 
 async def enrich_issue(
-    session: aiohttp.ClientSession, issue: Dict[str, Any], enrichments: List[Enrichment]
+    session: aiohttp.ClientSession,
+    issue: Dict[str, Any],
+    enrichments: List[Enrichment],
+    host: Optional[str] = SENTRY_HOST,
```

Comment:
This is slightly too broad. Since there's a `str`default value for this arg, and since we need a `host`, we can drop the `Optional`.

Reply:
Ah, I figured the default value was what allowed this to be `Optional`. Makes sense!